.changeset/gold-buses-thank.md

```diff
@@ -0,0 +1,18 @@
+---
+'xstate': major
```

Comment:
We should add a similar changeset for `@xstate/fsm` now and also somehow document the same change for `transition`, `raise`, `send`, `respond` etc

Reply:
I'll get on it